README.md

```diff
@@ -77,8 +77,14 @@ To install the Taipy stable release run:
 pip install taipy
 ```
 
-To install Taipy on a Conda Environment or from a source, please refer to the [Installation Guide](https://docs.taipy.io/en/latest/installation/).<br />
-To get started with Taipy, please refer to the [Getting Started Guide](https://docs.taipy.io/en/latest/getting_started/).
+
+Ready to Install Taipy? 🚀<br>
+Get everything set up in no time! Whether you're using a Conda environment or installing from source, follow our [Installation Guide](https://docs.taipy.io/en/latest/installation/) for step-by-step instructions.<br />
+
+Excited to Dive In? 💡 <br>
+Start building with Taipy today! Our [Getting Started Guide](https://docs.taipy.io/en/latest/getting_started/) is the perfect place to begin your journey and unlock the full potential of Taipy.
```

Comment:
The correct link to the Getting Started: https://docs.taipy.io/en/develop/tutorials/getting_started/

Reply:
@FlorianJacta I believe the link should point to latest, no?

Reply:
I am not sure but this link will be dead in some hours or days. I will say that it should be develop in develop and latest in latest